torch/distributed/pipeline/sync/pipe.py

```diff
@@ -269,18 +269,20 @@ class Pipe(Module):
     Example::
         Pipeline of two FC layers across GPUs 0 and 1.
 
-        >>> # Need to initialize RPC framework first.
-        >>> os.environ['MASTER_ADDR'] = 'localhost'
-        >>> os.environ['MASTER_PORT'] = '29500'
-        >>> torch.distributed.rpc.init_rpc('worker', rank=0, world_size=1)
-        >>>
-        >>> # Build pipe.
-        >>> fc1 = nn.Linear(16, 8).cuda(0)
-        >>> fc2 = nn.Linear(8, 4).cuda(1)
-        >>> model = nn.Sequential(fc1, fc2)
-        >>> model = Pipe(model, chunks=8)
-        >>> input = torch.rand(16, 16).cuda(0)
-        >>> output_rref = model(input)
+        # Need to initialize RPC framework first.
```

Comment:
do we need an additional indent to make it render as code?

Reply:
Add the indent. Thanks for the reminder!